README.md

```diff
@@ -46,12 +44,13 @@ check [server](#server) config for more details.
 
 ### ActionLogger
 
-`actionLogger(logger)` is a redux middleware that adds redux actions in the logster's buffer as `info` level logs.
-actionFilters and stateFilters and be applied to filter out the is logged. [see more](#filters)
+`actionLogger(logger [,actionFilter, stateFilter, level])` is a redux middleware that adds redux actions in the logster's buffer as `info` level logs.
```

Comment:
a very good use case would be to use different state filters based on different actions, something like:
```
case 'ADD_TO_CART':
return {
    products: state.products
}
case 'USER_LOGGED_IN':
return {
  user: state.user
}
```
what do you think?

Reply:
That is a very good use case, let's create a issue and work on this.